packages/data-service/src/data-service.ts

```diff
@@ -1488,6 +1520,28 @@ class DataServiceImpl extends WithLogContext implements DataService {
     return (await coll.findOneAndUpdate(filter, update, options)).value;
   }
 
+  @op(mongoLogId(1_001_000_183))
```

Comment:
Did the script suggest this number? It should always be suggesting one that's higher than all other used numbers

Reply:
It was, I think the problem is that I merged the main a couple of times and for this actually, it was fine, but for replaceOne 1_001_000_184 was errored as a duplicate and I keep changing it.